docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -27,3 +27,9 @@ Check out the [Getting started with Serverless guide](/dagster-cloud/getting-sta
 With a [Hybrid Deployment](/dagster-cloud/deployment/hybrid), your Dagster code is executed in your infrastructure and requires setting up an agent for use with Dagster Cloud. The agent runs in your infrastructure, executes your code, and streams metadata about code execution over HTTPS back to Dagster Cloud.
 
 Check out the [Getting started with Hybrid guide](/dagster-cloud/getting-started/getting-started-with-hybrid-deployment) to get up and running.
+
+---
+
+## Environment variables and secrets
+
+Once you're up and running with a Serverless or Hybrid deployment, the next step is (probably) setting up connections in your Dagster code. Check out the [Environment variables and secrets in Dagster Cloud guide](/dagster-cloud/developing-testing/environment-variables-and-secrets) to learn how to use environment variables to securely pass secrets in Dagster.
```

Comment:
i'm not sure 'connections' is a universal term in the way its being used here - Maybe 'connections to other services' or something